Netkan/Program.cs

```diff
@@ -106,13 +106,13 @@ public static int Main(string[] args)
             }
             catch (Exception e)
             {
-                e = e.GetBaseException() ?? e;
+                e = e.GetBaseException();
 
                 Log.Fatal(e.Message);
 
                 if (Options == null || Options.Debug)
                 {
-                    Log.Fatal(e.StackTrace);
+                    Log.Fatal(e.ToString());
```

Comment:
Can we keep the stack trace? It might not be included in every implementation of `ToString`.

Reply:
According to [this section](https://docs.microsoft.com/en-us/dotnet/api/system.exception.tostring?view=netframework-4.8#remarks) it is by default. Except `ToString()` is overridden in one of the krakens...

Reply:
Can we just leave it as it was? We already print `e.Message` above, so I'm not sure why we needed a change here.

Reply:
Right, I think I did this because I missed to set `kraken.message` in the constructors of the new `NativeAndCurlDownloadFailedKraken`, that's why I made the custom `ToString()` and changes this line here.

I added later that the message is actually set in the constructor, so it should be fine indeed if I revert this. Thanks!